packages/react-router/tests/link.test-d.tsx

```diff
@@ -68,33 +68,183 @@ const routeTree = rootRoute.addChildren([
   indexRoute,
 ])
 
-type RouteTree = typeof routeTree
+const defaultRouter = createRouter({
+  routeTree,
+})
+
+const routerAlwaysTrailingSlashes = createRouter({
+  routeTree,
+  trailingSlash: 'always',
+})
+
+const routerNeverTrailingSlashes = createRouter({
+  routeTree,
+  trailingSlash: 'never',
+})
+
+const routerPreserveTrailingSlashes = createRouter({
+  routeTree,
+  trailingSlash: 'preserve',
+})
+
+type DefaultRouter = typeof defaultRouter
+
+type RouterAlwaysTrailingSlashes = typeof routerAlwaysTrailingSlashes
+
+type RouterNeverTrailingSlashes = typeof routerNeverTrailingSlashes
+
+type RouterPreserveTrailingSlashes = typeof routerPreserveTrailingSlashes
 
 test('when navigating to the root', () => {
-  expectTypeOf(Link<RouteTree, string, '/'>)
+  type hi = DefaultRouter['options']['trailingSlash']
```

Comment:
remove?

Reply:
Oops. Ye. I'm changing the type tests atm